tests/data/cases/pep_572_py39.py

```diff
@@ -5,3 +5,7 @@
 # We better not remove the parentheses here (since it's a 3.10 feature)
 x[(a := 1)]
 x[(a := 1), (b := 3)]
+
+# don't remove the brackets here, it changes the meaning of the code.
```

Comment:
This test seems like the wrong place, it's about PEP 572 (walrus expressions). Maybe move it to one of the `context_managers_*` tests.